script/openqa-bootstrap

```diff
@@ -46,7 +46,12 @@ if ping -c1 download.suse.de. && (! rpm -q ca-certificates-suse) ; then
     if ! zypper info ca-certificates-suse | grep -q ':' ; then
         # add suse ca repo if needed
         # use this way of adding the repo to be distro agnostic
-        zypper -n addrepo obs://SUSE:CA SUSE:CA
+        if [ "$NAME" = "openSUSE Leap" ]; then
+            # avoid using `obs://…` URL to workaround https://bugzilla.opensuse.org/show_bug.cgi?id=1187425
+            zypper -n addrepo "https://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_${VERSION}" 'SUSE:CA'
+        else
+            zypper -n addrepo obs://SUSE:CA SUSE:CA
```

Comment:
`SUSE:CA` is not in obs but on ibs, so this would just have added the non existent obs repo.
I wonder if this ever worked.

```suggestion
            zypper -n addrepo http://download.suse.de/ibs/SUSE:/CA/openSUSE_Tumbleweed/ SUSE:CA
```

Reply:
It should have worked before. The error one gets under Leap 15.3 is:

```
Repository 'SUSE:CA' is invalid.
[SUSE:CA|http://download.suse.de/ibs/SUSE:/CA/Leap_15.3] Valid metadata not found at specified URL
```

So it is actually somehow clever enough to use `download.suse.de` as domain.

Reply:
Since it is clever enough I'll keep it this way. Keeping the `obs://`-URL in the `else` branch has the advantage that we don't have to handle different distribution names.

Reply:
That's what I thought too - after all I wrote that line initially I guess.

But:
```
# podman run -it --rm opensuse/tumbleweed bash
d6963ac04377:/ # zypper -n addrepo obs://SUSE:CA SUSE:CA
Guessed: platform = openSUSE_Tumbleweed
Adding repository 'SUSE:CA' ................................................................................................................................................................................[done]
Repository 'SUSE:CA' successfully added

URI         : https://download.opensuse.org/repositories/SUSE:/CA/openSUSE_Tumbleweed
Enabled     : Yes
GPG Check   : Yes
Autorefresh : No
Priority    : 99 (default priority)

Repository priorities are without effect. All enabled repositories share the same priority.
```

Reply:
And I can't find code in https://github.com/openSUSE/zypper/blob/master/src/utils/misc.cc#L504 that would do that.
In fact the zypper sourcecode doesn't contain the string `download.suse.de`

Reply:
Well never mind - I knew that back then and replace the url afterwards:
https://github.com/os-autoinst/openQA/blob/1fe2f6888265b99f7b6c4d8d64bb0f7b021320db/script/openqa-bootstrap#L55